SegmentRecord.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I don't think you meant to commit this file?

Reply:
Is this not the right format for a JSON file?

Reply:
The format looks fine, but we shouldn't upload data files Prometheus is generating to the repo. If you want/need a file for use in tests, we have specific directories for those.

Reply:
Ah I see